django/db/backends/sqlite3/base.py

```diff
@@ -324,11 +324,17 @@ class SQLiteCursorWrapper(Database.Cursor):
     def execute(self, query, params=None):
         if params is None:
             return Database.Cursor.execute(self, query)
+        if isinstance(params, (tuple)):
+            if params:
+                if isinstance(params[0], (int, long)):
```

Comment:
`long` doesn't exist on Python 3. You would want to use `six.integer_types` but as Simon said, this fix doesn't look elegant.